environment.yml

```diff
@@ -0,0 +1,184 @@
+name: usb
```

Comment:
what is usb?

Reply:
It is the name of the environment used by me. Can be assigned a project-specific name. Noted it.